sonic-slave/Dockerfile

```diff
@@ -242,6 +242,10 @@ RUN export VERSION=1.8.3 \
  && echo 'export GOROOT=/usr/local/go' >> /etc/bash.bashrc \
  && echo 'export PATH=$PATH:$GOROOT/bin' >> /etc/bash.bashrc
 
+# For go telemetry build
+RUN wget http://ftp.us.debian.org/debian/pool/main/g/golang-glide/golang-glide_0.12.3-2+b3_amd64.deb \
+ && dpkg -i golang-glide_0.12.3-2+b3_amd64.deb
```

Comment:
can we just use apt-get to installer?

Reply:
It's failed to use apt-get to install it. Because the glide apt-repository has no appropriate package for debian jessie/stretch.

```
$ sudo add-apt-repository ppa:masterminds/glide && sudo apt-get update
You are about to add the following PPA to your system:
 Glide dist
See https://glide.sh
 More info: https://launchpad.net/~masterminds/+archive/ubuntu/glide
Press [ENTER] to continue or ctrl-c to cancel adding it

gpg: keyring `/tmp/tmplu9zsuem/secring.gpg' created
gpg: keyring `/tmp/tmplu9zsuem/pubring.gpg' created
gpg: requesting key 890C81B2 from hkp server keyserver.ubuntu.com
gpg: /tmp/tmplu9zsuem/trustdb.gpg: trustdb created
gpg: key 890C81B2: public key "Launchpad PPA for Masterminds" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
OK

......

Err http://ppa.launchpad.net jessie/main amd64 Packages                                                                                    
  404  Not Found
Get:23 http://debian-archive.trafficmanager.net jessie/contrib amd64 Packages [59.2 kB]                                                    
Get:24 http://debian-archive.trafficmanager.net jessie/non-free amd64 Packages [101 kB]    
Fetched 31.0 MB in 4s (6421 kB/s)                                                         
W: Size of file /var/lib/apt/lists/debian-archive.trafficmanager.net_debian-security_dists_jessie_updates_main_binary-amd64_Packages.gz is not what the server reported 645826 645898
W: Size of file /var/lib/apt/lists/security.debian.org_dists_jessie_updates_main_binary-amd64_Packages.gz is not what the server reported 645826 645898
W: Failed to fetch http://ppa.launchpad.net/masterminds/glide/ubuntu/dists/jessie/main/binary-amd64/Packages  404  Not Found
$
$ sudo apt-get install glide
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package glide
```

Reply:
https://packages.debian.org/stretch/golang-glide

Reply:
why use ppa?

Reply:
then please back port golang-glide to jessie, or upgrade the telemetry docker to stretch.

Reply:
directly pull the golang-glide from stretch distribution in jessie docker is not a standard way.

maybe we should just do 'curl https://glide.sh/get | sh' as suggested by the glide website.